### PR TITLE
Issue #SB-28693 fix: contentType reference in contentapi.yml

### PR DESCRIPTION
**Pull request by Jayaprakash8887**

### CODE REVIEW

**Comment by amorphous-1** [comment]
Hello @Jayaprakash8887,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://933.qa.docs.sunbird.org/4.8.0/ .<br/> **Note:** Only _4.8.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://933.qa.docs.sunbird.org/4.8.0/apis/content// <br /> - http://933.qa.docs.sunbird.org/4.8.0/apis/contentapi// <br />  <br/> **Modified sections are -** <br />

File: automationScripts/spec.yml [hunk 1]
```diff
@@ -3589,6 +3589,68 @@ paths:
...
       security:
         - bearer: []
+  /dialcode/v2/read:
+    post:
+      summary: Get DIAL Code
+      parameters:
+        - name: X-Channel-ID
+          in: header
+          description: >-
+            It the Unique ID to identify the root organization to which the user belongs
```
Comment (anisha147) [inline automationScripts/spec.yml:3599]: Line 3599: Please add "is" in the description to make the sentence grammatically correct.

File: automationScripts/spec.yml [hunk 2]
```diff
@@ -3715,6 +3777,77 @@ paths:
...
+        - name: dialcode
+          in: path
+          description: The dialcode identifier which you want to update with context information
+          required: true
+          schema:
+            type: string
+        - name: X-Channel-ID
+          in: header
+          description: >-
+            It the Unique Id to identify the root organization to which the user belongs
```
Comment (anisha147) [inline automationScripts/spec.yml:3793]: Line 3793: please add "is" in the description.

File: apis/content/contentapi.yaml [hunk 3]
```diff
@@ -1842,7 +1842,7 @@ paths:
   /content/v1/search:
     post:
       description: |-
-        This API is associated with Searching Object created as part of sourcing object creation on the Sunbird Platform.
+        This API is associated with Searching content and collection objects created as part of sourcing object creation on the Sunbird Platform.
```
Comment (anisha147) [inline apis/content/contentapi.yaml:1845]: 1. Line 314, 315: Please rectify the formatting of the bullet points. It is appearing as sub-bullets.
2. Global: Please check the punctuation in the line “The Content Type entity is the media type of the resource.Possible media types can be”
3. Global: Please rectify the casing of “To” in the line “Please append a valid Content Id To the Request URL”
4. Line 6: It should be “Sunbird Platform. APIs perform”
5. Line 1848 & 2355: Please rectify the formatting of the line. It is appearing along with the bullet point.
6. Line 1028, 1623: Please rectify the formatting in the line for the success response.
7. Global: Please rectify the punctuations and spacings in all the response descriptions.

File: apis/contentapi/contentapi.yaml [hunk 4]
```diff
@@ -1822,7 +1822,7 @@ paths:
   /content/v1/search:
     post:
       description: |-
-        This API is associated with Searching Object created as part of sourcing object creation on the Sunbird Platform.
+        This API is associated with Searching content and collection objects created as part of sourcing object creation on the Sunbird Platform.
```
Comment (anisha147) [inline apis/contentapi/contentapi.yaml:1825]: 1. Line 314, 315, 438, &439: Please rectify the formatting of the bullet points. It is appearing as sub-bullets.
2. Global: Please check the punctuation in the line “The Content Type entity is the media type of the resource.Possible media types can be”
3. Global: Please rectify the casing of “To” in the line “Please append a valid Content Id To the Request URL”
4. Line 441: Please recheck the sentence formation
5. Line 1362, 1684, & 2224: Please insert a space before endpoint in “**endpoint executes request”
6. Line 1496: remove single tick at the end of the line
7. Line 398: Rectify the punctuations in line 398
8. Line 2322, 2698, 2996: Please change the casing of Url to URL
9. Line 1026, 1531: Please rectify the formatting in the line success response.
10. Global: Please rectify the punctuations and spacings in all the response descriptions.

File: apis/content/contentapi.yaml (hunk 3 above)
Comment (Jayaprakash8887) [reply apis/content/contentapi.yaml:1845]: @anisha147 Not feasible to address punctuations and spacings in all the response descriptions now. We can look into this gradually. However, I have tried to address issues at places similar to the line numbers you have mentioned.

File: apis/contentapi/contentapi.yaml (hunk 4 above)
Comment (Jayaprakash8887) [reply apis/contentapi/contentapi.yaml:1825]: @anisha147 Not feasible to address punctuations and spacings in all the response descriptions now. We can look into this gradually. However, I have tried to address issues at places similar to the line numbers you have mentioned.

**Comment by amorphous-1** [comment]
Hello @Jayaprakash8887,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://933.qa.docs.sunbird.org/4.8.0/ .<br/> **Note:** Only _4.8.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://933.qa.docs.sunbird.org/4.8.0/apis/content// <br /> - http://933.qa.docs.sunbird.org/4.8.0/apis/contentapi// <br />  <br/> **Modified sections are -** <br />

**Comment by amorphous-1** [comment]
Hello @Jayaprakash8887,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://933.qa.docs.sunbird.org/4.8.0/ .<br/> **Note:** Only _4.8.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://933.qa.docs.sunbird.org/4.8.0/apis/content// <br /> - http://933.qa.docs.sunbird.org/4.8.0/apis/contentapi// <br />  <br/> **Modified sections are -** <br />

**Comment by amorphous-1** [comment]
Hello @Jayaprakash8887,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://933.qa.docs.sunbird.org/4.8.0/ .<br/> **Note:** Only _4.8.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://933.qa.docs.sunbird.org/4.8.0/apis/content// <br /> - http://933.qa.docs.sunbird.org/4.8.0/apis/contentapi// <br />  <br/> **Modified sections are -** <br />

**Pull request merged**